src/main/java/nl/cherement/jak/service/BoardService.java

```diff
@@ -17,25 +16,21 @@
     BoardRepository boardRepository;
 
     @Autowired
-    UserRespository userRespository;
+    UserService userService;
 
     public BoardService(BoardRepository repository) {
         super(repository);
     }
 
-    public List<BoardEntity> findByUserName(String username) {
+    public List<BoardEntity> findBoardByUserName(String username) {
```

Comment:
Waarom de board in de naam, het is duidelijk omdat het in de boardService zit en de method verwacht een BoardEntity terug.